React/nested-event-handling.md

```diff
@@ -190,11 +260,14 @@ markPresent = (studentIndex) => {
 
 **Question:**  What way do we have to pass data from a parent component to a child component.
 
-If you answered `props` you're correct!
+<details>
+<summary>Answer</summary>
```

Comment:
Could you make the `**Question:**` line be the summary? You'll have to replace the asterisks with a `<b>`.